src/components/CreateTargetForm/styles.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from 'react-native';
+import { BLACK, BACKGROUND } from 'constants/colors';
+import typography from 'constants/typography';
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    position: 'absolute',
```

Comment:
What is the intended behaviour of this styles?

Reply:
Hi! This was done to use it as a layer that covers all the screen so that the user can tap outside the form to hide it, as it can be seen in the gif.